test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -35,6 +35,43 @@ Describe "Get-ChildItem" -Tags "CI" {
             $files[2].Name     | Should Be "B"
             $files[3].Name     | Should Be "c"
             $files[4].Name     | Should Be "D"
+            # Test data cleanup
+            Remove-Item -Path $TestDrive -Recurse -Force
+        }
+
+        It "Should list hidden files as well when 'Force' parameter is used" {
+            $TestFile1 = join-path $TestDrive ".TestFile1"
+            $TestFile2 = join-path $TestDrive "TestFile2"
+            New-Item -Path $TestFile1 -ItemType "File" -Force | %{$_.Attributes = "hidden"}
+            New-Item -Path $TestFile2 -ItemType File
+
+            $files = Get-ChildItem -path $TestDrive -Force
+            $files | Should not be $null
+            $files.Count | Should be 2
+            $files[0].Name | Should be ".TestFile1"
+            $files[1].Name | Should be "TestFile2"
+            # Test data cleanup
+            Remove-Item -Path $TestDrive -Recurse -Force
+        }
+
+        It "Should list only hidden files when 'Hidden' parameter is used" {   
+            $TestFile1 = join-path $TestDrive ".TestFile1"
+            $TestFile2 = join-path $TestDrive "TestFile2"
+            New-Item -Path $TestFile1 -ItemType "File" -Force | %{$_.Attributes = "hidden"}
+            New-Item -Path $TestFile2 -ItemType File
```

Comment:
These 4 statements seems redundant for each test. Maybe you should put them in a `BeforeAll` or `BeforeEach` block.